charts/ingress-nginx/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: ingress-nginx
 # When the version is modified, make sure the artifacthub.io/changes list is updated
 # Also update CHANGELOG.md
-version: 3.21.0
+version: 3.22.0
```

Comment:
Please do not update the version. I need to add other PRs.

Reply:
You need to add other PRs to a chart version?  Is the workflow for this project to let the chart releaser action fail and then a member will update the version when they want to release the chart?  Do you want me to push another commit to change the version back to 3.21.0?

Reply:
> Do you want me to push another commit to change the version back to 3.21.0?

Yes, and squash the commits, please.

Reply:
I am not where I can squash them right now.  I will have to get to that later

Reply:
Squashed and rebased